README.md

```diff
@@ -2,57 +2,155 @@
 
 GOG Galaxy 2.0 Community integration for Steam.
 
-## Installation
+## Open Beta:
+This project is in open beta. It is not completely bulletproof, but it has been extensively tested. We are hoping that crowdsourcing the error checking will help find anything we may have missed.
+
+### Known Issues:
+* Large libraries are known to be a little wonky. Sometimes, the plugin may crash after initially starting it, but then immediately work properly once you hit "retry". There's not much we can do here.
+* If you take too long to enter a 2FA code, any login stuff after that have a small chance of crashing. This is because steam kicks us out after roughly a minute of inactivity. We reconnect, but if we were sending them something when they kicked us out, we'd never get a response and after a minute of waiting, the plugin will close. Restarting the plugin will fix this, but it is something we are looking at fixing in a later version. In our tests, it has happened, but is difficult to replicate. If this issue is more prevalent than we anticipate, it's something we will address immediately. 
+* Closing GOG immediately after connecting for the first time may cause GOG to crash. Similarly, disconnecting Steam immediately after connecting it may cause the plugin to crash, though you usually can hit retry and it'll work again. Basically, GOG thinks we are immediately done syncing once we connect, and that is not the case, especially for larger libraries. So, it thinks it can disconnect immediately too, and if we're in the middle of writing some data down, it has to wait, but has basically hamstrung us in the process. It's not an ideal situation. Fortunately, it only happens once, and won't happen at all if you give it time.
+
+### Installation:
+* There is a zip file in the releases directory. Download this.
+* Navigate to where GOG stores the steam plugin. 
+  * Windows:
+    <br>`%localappdata%\GOG.com\Galaxy\plugins\installed\steam_ca27391f-2675-49b1-92c0-896d43afa4f8`
+  * MacOS:
+    <br>`~/Library/Application Support/GOG.com/Galaxy/plugins/installed/steam_ca27391f-2675-49b1-92c0-896d43afa4f8`
+* If the file does not exist, create it. If it does, delete everything inside it. 
+* Extract the zip release so all the contents are in that file. 
+* Start GOG Galaxy. 
+
+### Logging: 
+We tried to kill as many bugs and test as many behaviors as possible, but we aren't perfect.
+<br>You may find some case we haven't tested.
+<br>Please raise an issue here, and in the comment, attach your logs.
+<br>They can be found here: 
+* Windows:<br>`%programdata%\GOG.com\Galaxy\logs`
+* MacOS:  <br>`/Users/Shared/GOG.com/Galaxy/Logs`
+
+We typically only need the `steam_<numbers and letters>.log` file.
+
+## Setup (For Developers)
+* Download [Python 3.7.9 32-bit][Python379]
```

Comment:
3.7.9 is the last patch for 3.7 that's easily available. Anything newer you need to build yourself. I didn't realize gog didn't update their 3.7 instance since 3.7.3, which they *really* should. 3.7.x changes are just supposed to be patches so you get the latest security fixes, so they shouldn't affect anything.

Reply:
I can update the Readme and note that "gog uses 3.7.3, that this shouldn't affect anything but if you want to ensure there won't be any surprises you can use that" or something similar.

Reply:
It would be  more than enough enough to to use 3.7.3 in CI. Don't bother with chaning Readme. I'm resolving this treat